src/DI/PhpReflection.php

```diff
@@ -81,6 +87,9 @@ public static function expandClassName($name, \ReflectionClass $rc)
 		if (empty($name)) {
 			throw new Nette\InvalidArgumentException('Class name must not be empty.');
 
+		} elseif (in_array($name, PHP_VERSION_ID >= 70000 ? ['string', 'int', 'float', 'bool', 'array', 'callable'] : ['array', 'callable'], TRUE)) {
```

Comment:
Yes, `string` is valid class in PHP < 7, but it is very unlikely that `@return string` in PHP < 7 means class named `string`.